index.html

```diff
@@ -51,6 +51,114 @@ <h1 class="my-name" >I'm Omotoyosi. <br>
                 </div>
             </div>
         </section>
-    </div>
+    </section>
+    <section class="second-section">
+        <div class="item works-1">
+            <img src="images/works-1.svg" alt="template-image" srcset="">
+            <p class="title-1">Tonic</p>
```

Comment:
- [ ] Your project cards are great, but the title of each project card should be wrapped with a proper `hx` tag like an `h3` tag for better HTML semantics. Kindly change each title tag for this reason.
 - [ ] Kindly change the `line-height` property to match the Figma template project card title, currently there is a huge gap between the lines and it would be better without it.
 - Your design:
 - 
<img width="312" alt="image" src="https://user-images.githubusercontent.com/61919617/227254084-6ef45424-4bfd-4235-84d1-8e388fa845c5.png">